en/2-build-in-accessibility-from-start.md

```diff
@@ -9,6 +9,41 @@ altLangPage: 2-integrer-accessibilite-des-depart
 
 **[TODO: Add/revise introductory text]**
 
+Building in accessibility from the start is key to ensure that your programs, service, tools or application can be used by everyone, including those living with a permanent disability, but also by those who may have a temporary limitation or disability due to illness, accident, environmental changes or technological difficulties.
+
+Worldwide, Over a billion people, about 15% of the world's population, have some form of disability. Between 110 million and 190 million adults have significant difficulties in functioning. (<a href="http://www.who.int/news-room/fact-sheets/detail/disability-and-health">World Health Organization</a>)
+
+In Canada:
+
+- 14% of the population (4.9 million Canadians) identify as a person with a disability
+- 30% of the population have a disability if you include “invisible” disabilities (colour blindness, cognitive, mental health or chronic pain related)
+- 50% of the population have a disability if you take into account age-related impairments (low vision, low hearing, cognitive impairments),
+- At least once in a person’s lifetime, they also may have a temporary disability brought on through accident, illness, repetitive strain, lifecycle changes (pregnancy).
+
+**Curb Cut Effect**
+
+Curb cuts are intended to help wheelchair get on sidewalks, but they also help bicyclists, parents with strollers, delivery people, and many other nondisabled groups. This benefit to others became known as the “Curb-Cut Effect”.
+
+When inclusive and accessible design is in place, it is a benefit for all and seamlessly meets the needs of individuals across the board, including people without disabilities.
+
+**Situational Limitations**

```

Comment:
Should this be a heading? If so, use three number signs at the start (removing the asterisks) to make it a level 3 heading. If not, then suggest adding a colon.